tests/testthat/test-helpers.R

```diff
@@ -17,3 +17,10 @@ test_that('tictoc', {
 test_that('log with separator', {
     expect_output(log_with_separator(42), '===')
 })
+
+test_that('log failure', {
+  expect_output(log_failure("foobar"), NA)
+  expect_output(log_failure(foobar), 'ERROR.*foobar')
```

Comment:
I think needs to be wrapped in `try` not to fail this test:

```suggestion
  expect_output(try(log_failure(foobar), silent = TRUE), 'ERROR.*foobar')
```